### ISSUE TITLE
CL_BUILD_PROGRAM_FAILURE

### ISSUE THREAD

**Issue by easysoft2k15** [uncomplicate/bayadera#3]
I tried the 

> complete_example_test.clj 

but there seems to be a problem with OpenCL:

```
(analysis)
ExceptionInfo OpenCL error: CL_BUILD_PROGRAM_FAILURE.  clojure.core/ex-info (core.clj:4617)
```

If i try:

`(display-sketch)`

I get a blank page.
Moreover could You please make Bayadera available into Clojars?
Thank You

**Comment by blueberry** [comment]
Do you have an OpenCL 2.0 capable hardware?

**Comment by easysoft2k15** [comment]
I've a Dell Inspiron laptop with i7 processor and
1. AMD Radeon R5 M335
2. Intel HD Graphics 520
Is there some documentation i can refer to to understand what i need to run Bayadera library?
Thank You for the super fast replay!

**Comment by blueberry** [comment]
Currently you need an OpenCL 2.0 GPU (Amd preferred) and Intel MKL installed (see neanderthal getting started guide). You can check which version of OpenCL is supported by your GPU by running clinfo in the terminal. Anyway, don't expect great speed from R5.

**Comment by blueberry** [comment]
btw. a good read to understand how to setup the gpu and run gpu-enabled code is in the guides at http://clojurecl.uncomplicate.org

**Comment by easysoft2k15** [comment]
1. Amd GPU is OpenCL 1.2 AMD-APP (2079.5)
2. Intel GPU is OpenCL 2.0 (Build 2)
The Amd is listed first at the clinfo command prompt. Do I need to instruct Bayadera to use the Intel GPU?
Right now I'm only interested in exploring Bayadera and get a feel on how to use it and don't care to much about performance.
Once again thank You for Your time.

**Comment by blueberry** [comment]
Yes, you need to instruct bayadera to use the right platform (if more than one is there) and the desired device on that platform. Se how to do that in ClojureCL guides and test examples.
